lib/components/molecules/Accordion/Accordion.js

```diff
@@ -0,0 +1,107 @@
+// @flow
+/**
+ *
+ * Accordion
+ *
+ */
+import React, { useState } from 'react';
+import type { Node } from 'react';
+import styled from 'styled-components';
+import classnames from 'classnames';
+import styles from './Accordion.style';
+import type { AccordionHeaderProps, AccordionPanelProps, AccordionProps } from './types';
+
+const AccordionHeader = ({
+  children,
+  className,
+  ariaLabelAttr,
+  accordionPanelId,
+}: AccordionHeaderProps): Node => {
+  const [isExpanded, toggleExpanded] = useState(false);
+
+  const handleClick = () => toggleExpanded(!isExpanded);
+
+  return (
+    <summary
+      className={className}
+      role="button"
+      aria-expanded={isExpanded}
+      aria-label={ariaLabelAttr}
+      aria-labelledby={accordionPanelId}
+      onClick={handleClick}
+      onKeyPress={handleClick}
```

Comment:
onKeyPress={handleClick} is wrong. You must handle which keys it will respond to